src/sinks/http.rs

```diff
@@ -18,9 +18,25 @@ use hyper::{Body, Client, Request};
 use hyper_tls::HttpsConnector;
 use indexmap::IndexMap;
 use serde::{Deserialize, Serialize};
+use snafu::{ResultExt, Snafu};
+use std::error::Error;
 use std::time::Duration;
 use tower::ServiceBuilder;
 
+#[derive(Debug, Snafu)]
+enum BuildError {
+    #[snafu(display("{}: {}", source, name))]
```

Comment:
Should these say something about them being invalid? Or do we get the name included automatically?

Reply:
The `source` is either `http::header::InvalidHeaderName` or `::InvalidHeaderValue` which should say that.

Reply:
Oh, obviously. Sorry, totally misread that :expressionless: